### PR TITLE
new icon: denojs (original, original-wordmark, plain, plain-wordmark)

### PR DESCRIPTION
**Pull request by informeai**
added deno icon and resolve problem in issue #260

### CODE REVIEW

**Comment by amacado** [comment]
Hello @informeai! Thanks for picking up denojs.
I have to tell you, that this sadly does not fulfill our requirements.

1. Instead of a vector graphic you have created a svg containing a base64 encoded `image/png`
2. You added `plain` and `plain-wordmark` in the devicon.json, but you provided those icons not in your pull request
```
"font": [
   "plain",
   "plain-wordmark"
]
```

Maybe you can have a look at our [CONTRIBUTING.md](https://github.com/devicons/devicon/blob/master/CONTRIBUTING.md) and try reworking `denojs` icon?

File: devicon.json [hunk 1]
```diff
@@ -888,6 +888,26 @@
...
+            "language"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark"
+            ],
+            "font": [
+                "plain",
+                "plain-wordmark"
```
Comment (Panquesito7) [inline devicon.json:905]: The icon has one color already.
```suggestion
                "original",
                "original-wordmark"
```

File: devicon.json [hunk 2]
```diff
@@ -888,6 +888,26 @@
...
+            ]
+        },
+        "color": "#000000",
+        "aliases": []
```
Comment (Panquesito7) [inline devicon.json:909]:
```suggestion
        "aliases": [
            {
                "base": "original",
                "alias": "plain"
            },
            {
                "base": "original-wordmark",
                "alias": "plain-wordmark"
        ]
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
I'm still working on making the `original-wordmark` version pure SVG.

File: icons/denojs/deno-original-wordmark.svg [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+</defs>
+</svg>
```
Comment (Panquesito7) [inline icons/denojs/deno-original-wordmark.svg:9]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" viewBox="0 0 128 128"><path style="stroke:none;fill-rule:evenodd;fill:#000;fill-opacity:1" d="M60.348.027c-.246.024-1.035.118-1.75.184C48.465 1.27 38.492 5.559 30.355 12.37c-1.5 1.25-4.925 4.668-6.171 6.172-6.73 8.047-10.672 17.074-12.125 27.758-.325 2.375-.325 10.379 0 12.754 1.453 10.683 5.394 19.715 12.125 27.761 1.246 1.504 4.671 4.922 6.171 6.168 8.043 6.739 17.079 10.672 27.762 12.13 2.371.32 10.38.32 12.75 0 10.688-1.458 19.719-5.391 27.766-12.13 1.5-1.246 4.922-4.668 6.168-6.168 6.73-8.046 10.672-17.078 12.12-27.761.329-2.375.329-10.38 0-12.754-1.448-10.684-5.39-19.711-12.12-27.758-1.246-1.504-4.668-4.922-6.168-6.172C90.609 5.664 81.493 1.676 70.945.261c-1.144-.16-2.797-.218-5.867-.245-2.344-.032-4.488-.008-4.73.011zm.648 5.598c0 1.512.098 4.805.246 7.93.078 1.527.164 3.715.207 4.855.145 4.043.57 14.652.637 15.602l.062.941-.582-.062c-.324-.024-.62-.102-.68-.141-.034-.055-.128-.945-.179-2-.23-4.61-.996-22.504-1.07-25.113l-.086-2.88.367-.05c.195-.031.52-.066.723-.082l.355-.012zm12.399-.313c.007.016.05 3.926.085 8.696.06 4.781.122 9.066.16 9.535.036.477.028.895-.038.918-.055.035-.375.035-.7 0l-.597-.05-.086-3.77c-.059-2.086-.121-4.395-.149-5.145-.097-2.293-.191-10.074-.113-10.27.05-.144.195-.16.734-.066.38.074.692.14.704.152zm-29.97 4.34c.083.114.688 7.258 1.438 16.598.25 3.176.492 6.078.535 6.438.063.656.051.687-.363.945-.234.144-.469.262-.52.262-.038 0-.113-.34-.156-.754-.183-1.688-.882-9.82-1.37-15.703a1185.2 1185.2 0 0 0-.587-6.891c-.062-.57-.05-.621.29-.8.41-.22.644-.263.734-.095zm35.669 2.606c.363.129.398.195.48.93.13 1.296.09 7.492-.05 7.492-.38 0-1.157-.442-1.22-.688-.042-.152-.074-1.992-.074-4.086 0-4.207-.058-3.96.864-3.648zm-23.93.61c0 .347.094 2.277.191 4.312.11 2.03.227 4.61.278 5.73.086 2.254.113 2.172-.754 2.227-.426.008-.426.008-.453-.64-.028-.36-.086-1.49-.172-2.52-.066-1.04-.215-3.282-.32-4.989-.106-1.715-.235-3.41-.274-3.773-.078-.649-.062-.672.285-.8.207-.08.567-.145.805-.145.414-.012.414-.012.414.597zm35.941 1.777.438.18.098 1.734c.035.945.058 2.996.035 4.539l-.035 2.82-.551-.273-.54-.258-.042-4.305c-.024-2.367-.016-4.371.012-4.469.05-.191.05-.191.585.032zM67.45 16.34c.078.074.258 6.36.274 9.543l.007 1.922-.605-.09c-.352-.043-.637-.078-.648-.094-.04-.023-.43-9.621-.43-10.558v-.961l.656.078c.383.05.707.117.746.16zm29.836.918c.156.18.18 3.02.223 18.668.035 17.972.035 18.437-.195 18.437a.8.8 0 0 1-.458-.175c-.195-.137-.218-1.704-.285-17.055-.05-9.301-.117-17.754-.152-18.774l-.082-1.867.375.274a3.4 3.4 0 0 1 .574.492zm-11.62 2.656c.062.05.128.785.136 1.633.07 2.863.078 20.418.015 20.469-.023.027-.261-.04-.515-.141l-.469-.195V19.64l.363.079c.196.05.403.129.47.195zm-35.763.727c.04.347.106 1.14.145 1.785.043.64.129 2.086.207 3.21.16 2.282.129 2.427-.606 2.427-.39 0-.441-.04-.496-.426-.132-.766-.586-7.14-.515-7.246.058-.09.738-.32 1.047-.348.082-.012.18.258.218.598zm-21.422 3.3c.137 1.375.422 4.258.63 6.387.222 2.137.402 3.953.402 4.027 0 .16-.996.649-1.102.551-.066-.066-1.36-11.535-1.36-12.05 0-.31.997-1.622 1.126-1.477.043.027.168 1.176.304 2.562zm50.797 1.364.375.215.012 2.804c.016 1.528.04 3.238.04 3.793.026 1.168-.118 1.356-.817 1.113l-.403-.14v-1.117c0-.618-.039-2.418-.09-4.012l-.078-2.879h.297c.157 0 .453.105.664.223zM34.16 30.328c.238 2.488.613 6.684.856 9.332l.445 4.793-.508.504-.488.504-.098-1.023c-.05-.57-.308-3.211-.582-5.883a1564.24 1564.24 0 0 1-.851-8.602l-.368-3.73.504-.465c.457-.426.512-.442.598-.207.05.129.273 2.281.492 4.777zm-9.7 1.57c.282 2.551 1.04 9.344 1.685 15.082.648 5.739 1.253 11.27 1.375 12.305.101 1.035.28 2.606.382 3.5.188 1.516.188 1.617-.02 1.774-.312.234-.44.203-.44-.067 0-.113-.118-1.101-.262-2.16-.23-1.777-.621-4.781-1.492-11.535a767.578 767.578 0 0 1-.774-6.02c-.258-1.957-.633-4.87-.844-6.48-.21-1.606-.554-4.352-.78-6.129-.231-1.762-.372-3.324-.333-3.492.09-.352.852-1.567.934-1.492.027.035.28 2.148.57 4.714zm-3.964 5.582c.195 1.56.559 4.395.805 6.32.242 1.923.554 4.317.695 5.317l.246 1.817-.273.23c-.149.129-.309.18-.36.129-.05-.05-.152-.57-.23-1.14-.078-.571-.563-3.903-1.078-7.391l-.922-6.344.297-.879c.16-.477.324-.867.367-.867.05 0 .246 1.266.453 2.808zm71.121-1.695c.188.18.211.98.211 6.93v6.726h-.36c-.788 0-.78.192-.831-7.168l-.035-6.695h.41c.222 0 .508.09.605.207zm-28.355 2.317c2.93.441 5.48 1.261 8.004 2.554 1.636.844 2.41 1.422 4.226 3.16 2.695 2.579 4.356 4.836 5.906 8.02 2.266 4.64 3.153 8.746 4.266 19.754.504 4.937 1.164 13.89 1.305 17.558.039 1.063.129 2.809.21 3.883.145 2.281.31 1.973-1.702 2.946-2.782 1.347-5.465 2.273-9 3.148-4.325 1.059-7.125 1.395-11.399 1.414l-3.11.012.024-1.492c0-.817.078-2.72.156-4.207.391-7.22.309-16.325-.191-21.38-.289-2.902-.844-6.429-1.156-7.202-.063-.172.23-.31 1.515-.75 2.348-.817 4.38-1.844 4.692-2.344.558-.961-.442-2.336-1.715-2.336-.215 0-.879.234-1.484.5-2.903 1.344-8.72 2.906-12.075 3.258-2.324.25-5.925.105-8.422-.352-1.363-.25-3.796-1.164-5.832-2.21-2.343-1.224-3.78-2.84-4.214-4.759-.235-1.039-.168-3.109.129-4.273.324-1.285 1.234-3.152 2.066-4.238 3.687-4.856 11.309-9.07 19.117-10.543 2.492-.47 6.012-.52 8.684-.121zm39.968.402c.47.195.52.246.52.687.023 3.536-.05 11.621-.121 11.957-.008.118-.68.13-.969.024-.18-.059-.207-.758-.207-6.477 0-4.09.05-6.398.129-6.398.066 0 .367.094.648.207zm5.66 7.117.47.195-.094 10.723c-.102 13.672-.118 14.07-.61 15.246-.8 1.934-.761 2.32-.683-6.148.023-4.239.093-8.528.117-9.528.027-.996.05-3.804.066-6.257 0-2.954.04-4.426.13-4.426.077 0 .347.09.605.195zm-77.913 3.227c.078 1.047.05 1.144-.325 1.976l-.418.883-.152-1.293c-.246-2.078-.246-2.168.246-2.477.223-.156.453-.257.492-.23.043.02.121.54.157 1.14zm-8.012 7.808c.12 1.063 1.027 8.106 1.496 11.57.7 5.348.75 6.114.437 5.79-.035-.04-.335-1.84-.648-4-2.02-13.973-2.008-13.88-1.84-13.98.352-.24.469-.11.555.62zm9.332 5.934c.078.324.453 3.832.453 4.265 0 .36-.465.7-.668.489-.066-.067-.238-1.168-.367-2.438-.145-1.277-.285-2.613-.324-2.965l-.075-.644.45.516c.257.285.488.632.53.777zm10.457 4.883c.14.129.234.644.348 2.047.195 2.523.218 2.332-.309 2.332-.547 0-.535.043-.711-2.243-.184-2.316-.184-2.293.164-2.293.172 0 .387.079.508.157zm10.258.191c.027.129.093.906.144 1.73.04.817.16 2.649.258 4.079.363 5.23.379 5.89.156 5.89-.246 0-.27-.187-.558-3.883a260.559 260.559 0 0 0-.426-5.351c-.113-1.41-.195-2.59-.152-2.617.136-.16.515-.04.578.152zM28.66 70.156c.074.114.563 4.262 1.176 9.922.184 1.711.36 3.356.398 3.664l.067.563-.32-.207c-.247-.172-.336-.34-.336-.637 0-.816-.56-6.191-.985-9.484-.25-1.868-.441-3.512-.441-3.668 0-.27.312-.375.441-.153zm74.766 5.41-.094 5.422-.324.473c-.738 1.094-.7 1.3-.656-4.762l.027-5.558.476-.496c.258-.262.512-.477.57-.477.04 0 .04 2.437 0 5.398zM91.44 84.16c.063 7.524.047 8.11-.156 8.344-.129.137-.246.234-.281.195-.094-.105-.223-16.52-.121-16.613.05-.05.18-.074.305-.059.164.035.203.957.253 8.133zm-42.31-5.156c.086.797.41 4.652.539 6.465.082 1.219.082 1.258-.207 1.347-.152.051-.332.04-.383-.027-.094-.09-.383-2.965-.726-7.398l-.098-1.059h.406c.375 0 .403.027.469.672zm-10.184 4.531c.086.094.153.418.153.742 0 .332.16 2.086.324 3.899.492 4.965.535 5.633.308 5.504-.335-.196-.632-.489-.57-.567.031-.05-.02-.703-.129-1.441a78.98 78.98 0 0 1-.258-2.719c-.05-.75-.168-2.129-.27-3.043-.245-2.27-.23-2.527.052-2.527a.63.63 0 0 1 .39.152zm6.13 8.723c.038.285.112 1.344.163 2.355.094 1.942.051 2.086-.476 1.688-.157-.121-.258-.614-.41-2.086-.274-2.672-.29-2.567.214-2.516.375.04.43.09.508.559zm0 0"/><path style="stroke:none;fill-rule:evenodd;fill:#000;fill-opacity:1" d="M39.484 44.492c-1.363.688-1.46 2.836-.183 3.602.933.547 1.87.281 2.355-.66.918-1.739-.574-3.75-2.172-2.942zm8.864 1.469c-1.254.89-1.254 3.059 0 3.96 1.246.864 2.851-.276 2.851-2.015 0-1.676-1.644-2.804-2.851-1.945zM9.629 99.086v20.434h3.094c3.515 0 6.199-.872 8.058-2.618 1.867-1.754 2.805-4.304 2.805-7.64 0-3.309-.93-5.825-2.785-7.559-1.86-1.746-4.547-2.617-8.078-2.617zm-8.64-6.523h9.093c5.078 0 8.871.37 11.363 1.101 2.492.723 4.621 1.945 6.399 3.676 1.578 1.512 2.746 3.254 3.508 5.234.757 1.969 1.144 4.196 1.144 6.688 0 2.531-.387 4.793-1.144 6.77-.762 1.984-1.93 3.726-3.508 5.234-1.79 1.734-3.942 2.964-6.461 3.695-2.508.723-6.274 1.078-11.301 1.078H.988zM63.02 113.64v2.305H44.504c.191 1.899.863 3.325 2.008 4.27 1.16.957 2.77 1.426 4.836 1.426 1.664 0 3.367-.25 5.117-.754 1.746-.5 3.535-1.262 5.367-2.282v6.243a34.088 34.088 0 0 1-5.613 1.613 28.49 28.49 0 0 1-5.61.562c-4.48 0-7.96-1.156-10.445-3.476-2.488-2.328-3.727-5.598-3.727-9.805 0-4.133 1.211-7.379 3.645-9.738 2.441-2.375 5.8-3.563 10.078-3.563 3.89 0 7.004 1.204 9.336 3.602 2.348 2.39 3.524 5.59 3.524 9.598zm-8.153-2.683c0-1.535-.437-2.77-1.308-3.707-.875-.95-2.024-1.426-3.442-1.426-1.531 0-2.773.442-3.726 1.32-.957.88-1.559 2.153-1.805 3.813zm39.664-.137v15.22h-8.176v-11.661c0-2.152-.05-3.629-.148-4.445-.098-.829-.273-1.434-.527-1.82a3.169 3.169 0 0 0-1.285-1.22c-.547-.292-1.172-.437-1.875-.437-1.7 0-3.032.652-4 1.945-.973 1.282-1.457 3.059-1.457 5.336v12.301H68.93v-25h8.132v3.66c1.235-1.457 2.54-2.531 3.922-3.226 1.375-.684 2.895-1.032 4.57-1.032 2.934 0 5.165.887 6.68 2.668 1.528 1.766 2.297 4.336 2.297 7.711zm18.742-4.597c-1.773 0-3.128.648-4.066 1.949-.93 1.289-1.39 3.14-1.39 5.57 0 2.422.46 4.274 1.39 5.574.938 1.29 2.293 1.926 4.066 1.926 1.743 0 3.075-.637 4.004-1.926.922-1.3 1.387-3.152 1.387-5.574 0-2.43-.465-4.281-1.387-5.57-.93-1.3-2.261-1.95-4.004-1.95zm0-5.782c4.313 0 7.68 1.184 10.102 3.54 2.434 2.347 3.648 5.605 3.648 9.761 0 4.149-1.214 7.406-3.648 9.766-2.422 2.344-5.79 3.515-10.102 3.515-4.332 0-7.718-1.171-10.168-3.515-2.433-2.36-3.648-5.617-3.648-9.766 0-4.156 1.215-7.414 3.648-9.762 2.45-2.355 5.836-3.539 10.168-3.539zm0 0"/></svg>

```

**Comment by informeai** [comment]
deno svg correction and devicon.json update

**Pull request closed without merging**